test/test_local_io.py

```diff
@@ -660,6 +660,16 @@ def test_io_path_file_lister_iterdatapipe(self):
         for path in datapipe:
             self.assertTrue(path in self.temp_sub_files)
 
+    @skipIfNoIoPath
+    def test_io_path_functional_list_files_iterdatapipe(self):
+        datapipe = IoPathFileLister(root=self.temp_sub_dir.name)
+
+        listed = list(datapipe.list_file_by_iopath())
```

Comment:
We can combine this test with the test above. And, for functional API, I suggest:
```suggestion
        datapipe = IterableWrapper([self.temp_sub_dir.name, ])
        liste = list(datapipe)
```

Reply:
450ac0ed71f4d8c1cea05daccbf959febcf0b06f